src/pages/page-router.ts

```diff
@@ -51,22 +55,40 @@ export class PageRouter extends LitElement {
       return html`<thank-you-page></thank-you-page>`;
     }
 
-    if (!this.recorder) {
+    if (this.showInstructions) {
       return html`
-        <consent-page
+        <instructions-page
           .wakeWord=${this.wakeWord}
           .description=${this.description}
-          .giveConsent=${(recorder: Recorder, description: string) => {
+          .startRecording=${(recorder: Recorder) => {
             this.recorder = recorder;
+
             recorder.addEventListener("stop", () => {
               this.wakeWord = undefined;
-              location.hash = "thank_you";
               this.recorder = undefined;
+              this.showInstructions = false;
+              location.hash = "thank_you";
             });
+
+            this.showInstructions = false;
+            this.recorder?.start();
+          }}
+        ></instructions-page>
+      `;
+    }
+
+    if (!this.recorder) {
```

Comment:
So this check is now a bit weird. The recorder is created inside instructions page. 

We might have to switch the router to just a state parameter now that we have multiple stages

Reply:
Hmm, I'm not sure how to best implement that